sktime/forecasting/compose/_fallback.py

```diff
@@ -36,6 +47,13 @@ class is useful in scenarios where the reliability of individual forecasting mod
     verbose : bool, default=False
         If True, raises warnings when a forecaster fails to fit or predict.
 
+    nan_predict_policy: str, default='ignore'
+        Determines the action to take if NaN values are found in the predictions. If set
+         to 'raise', the method raises a ValueError when NaN values are detected in the
```

Comment:
is this indentation change intented? Might render strangely.

Reply:
Good catch

Reply:
I mean, is the intentation indented, no, the intendation intented - well, you know what I mean

Reply:
I understood what you meant, the indentation was wrong. Was a good catch!

Reply:
I was just joking about the typo I made